File: INICIANTE/1115/main.dart
import 'dart:io';

void formattedPrint(int x, int y) {
  if (x > 0) {
    print(y > 0 ? 'primeiro' : 'quarto');
  } else {
    print(y > 0 ? 'segundo' : 'terceiro');
  }
}

void main() {
  var [x, y] = stdin.readLineSync()!.split(' ').map(int.parse).toList();
  while (x != 0 && y != 0) {
    formattedPrint(x, y);
    [x, y] = stdin.readLineSync()!.split(' ').map(int.parse).toList();
  }
}

File: INICIANTE/1115/main.js
function formattedPrint(x, y) {
    if (x > 0) {
        console.log(y > 0 ? 'primeiro' : 'quarto');
    } else {
        console.log(y > 0 ? 'segundo' : 'terceiro');
    }
}

let lines = require('fs').readFileSync('/dev/stdin', 'utf-8').split('\n');
let [x, y] = lines.shift().split(' ').map(Number);
while (x && y) {
    formattedPrint(x, y);
    [x, y] = lines.shift().split(' ').map(Number);
}
